src/kOS/kOS.csproj

```diff
@@ -201,7 +201,5 @@
   <Target Name="AfterBuild">
   </Target>
   -->
-  <ItemGroup>
```

Comment:
unrelated change that i keep meaning to make separate. its not needed anymore because the folder has files in it now